apps/api/src/app/integrations/e2e/create-integration.e2e.ts

```diff
@@ -452,7 +452,7 @@ describe('Create Integration - /integration (POST)', function () {
       providerId: EmailProviderIdEnum.SendGrid,
       channel: ChannelTypeEnum.EMAIL,
       active: true,
-      primary: false,
+      primary: true,
```

Comment:
This test was no longer accurate as the first integration will always be `primary` now. So we are updating the setup steps to make the first integration `primary`.